src/Widgets/Page.vala

```diff
@@ -108,13 +113,27 @@ namespace Network.Widgets {
                 try {
                     device.disconnect (null);
                 } catch (Error e) {
-                    warning (e.message);
+                    show_dialog (
+                        _("Failed To Disconnect"),
+                        _("Could not disconnect: %s.").printf (e.message),
```

Comment:
"Failed to Disconnect" and "Could not disconnect" are basically the same phrase. Probably don't need to repeat it :p